test/test_api.cpp

```diff
@@ -15,6 +15,7 @@
 #include <random>
 
 #include "test_utils.hpp"
+#if 0
```

Comment:
Are the tests broken? Or is this a leftover?

Reply:
a left over